### ISSUE TITLE
Disable autocomplete in column filters

### ISSUE THREAD

**Issue by billmn** [xaksis/vue-good-table#576]
What about disable autocomplete on column filters fields?

- [x] Enhancement Request

**Comment by xaksis** [comment]
@billmn not sure what you mean... there's no autocomplete on filters fields. you sure you're not talking about chrome's autocomplete?

**Comment by billmn** [comment]
Yes, I’m talking about browser’s autocomplete. Sorry, I didn’t specify it.

**Comment by xaksis** [comment]
hey @billmn, just out of curiosity how would you stop the browser based autocomplete on inputs? I've been researching a bit and haven't found a reliable way, autocomplete="off" is ignored now by chrome too. Have you done this before?

**Comment by billmn** [comment]
Yep, autocomplete="off" has been the standard for years ... but seems there's a bug regression on Chrome: https://bugs.chromium.org/p/chromium/issues/detail?id=914451

Seems that autocomplete="new-password" does the trick
